examples/frameworks/cherryapp.py

```diff
@@ -3,8 +3,12 @@
 cherrypy.config.update({'environment': 'embedded'})
 
 class Root(object):
+    @cherrypy.expose
     def index(self):
         return 'Hello World!'
-    index.exposed = True
 
-app = cherrypy.Application(Root(), script_name=None, config=None)
+cherrypy.config.update({'engine.autoreload.on': False})
```

Comment:
According to http://docs.cherrypy.org/en/latest/deploy.html#embedding-into-another-wsgi-framework, with `cherrypy.config.update({'environment': 'embedded'})` this line is not needed.

Could you also move line 3 here? Like

```py
cherrypy.config.update({'environment': 'embedded'})
cherrypy.server.unsubscribe()
cherrypy.engine.start()
app = cherrypy.tree.mount(Root())
```